yandexcloud/_helpers.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from yandex.cloud.iam.v1.service_account_service_pb2 import ListServiceAccountsRequest
+from yandex.cloud.iam.v1.service_account_service_pb2_grpc import ServiceAccountServiceStub
+from yandex.cloud.vpc.v1.network_service_pb2 import ListNetworksRequest
+from yandex.cloud.vpc.v1.network_service_pb2_grpc import NetworkServiceStub
+from yandex.cloud.vpc.v1.subnet_service_pb2 import ListSubnetsRequest
+from yandex.cloud.vpc.v1.subnet_service_pb2_grpc import SubnetServiceStub
+
+
+class Helpers(object):
+    def __init__(self, sdk):
+        self.sdk = sdk
+        self.client = sdk.client
```

Comment:
Isn't it redundant? Can we use just SDK here?

Reply:
Done